clar2wasm/src/tools.rs

```diff
@@ -0,0 +1,86 @@
+use clarity::{
```

Comment:
That's exactly what it is, just for testing. I used `developer-mode` because that is a feature name that other Stacks crates use (including `clarity`) for stuff that should be available in tooling (e.g. clarinet) but not in the stacks-node.

Reply:
Added comments in 9d74986.